src/components/CreateEmail.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import axios from 'axios'
+const emailTemplates = require('./EmailTemplates.js')
+
+const {
```

Comment:
you dont need to import these. The idea is that to make a request, they will do something like this:


```
import {PersonalPurchaseApproved} from "EmailTemplates.js"
import {sendEmail} from "createEmail.js"
const email = PersonalPurchasedApproved({issue: 12, reporter: bob})
sendEmail(email)
```